src/main/java/bwapi/EventHandler.java

```diff
@@ -30,10 +18,10 @@ public void operation(final ClientData.Event event) {
                 break;
             //case 3: //MenuFrame
             case SendText:
-                eventListener.onSendText(client.eventString(event.getV1()));
+                eventListener.onSendText(game.clientData().gameData().getEventStrings(event.getV1()));
```

Comment:
maybe we should add a `game.gameData()` method that calls `clientData().gameData()` internally to shorten some code/avoid duplication

Reply:
It's an option. I found myself mostly deleting methods like that because it was really easy to confuse the live game data with the potentially-delayed bot data. The fewer methods which returned one of those, the easier. Could go either way on it.

Reply:
ok, then maybe the method names should be more descriptive for the potentially-delayed bot data calls? But I'm also fine with leaving it like this

Reply:
Renaming Game.clientData() to "botClientData()" and Client.clientData() to "liveClientData()" to distinguish